spec/models/voter_spec.rb

```diff
@@ -0,0 +1,22 @@
+require "rails_helper"
+
+RSpec.describe Vote, type: :model do
+  subject { create(:vote) }
+
+  it { is_expected.to have_attribute(:value) }
+
+  it { is_expected.to belong_to(:voteable) }
+  it { is_expected.to belong_to(:voter) }
```

Comment:
might also be good to tack on the `class_name` and `foreign_key` options on here as well.

```ruby
it { is_expected.to belong_to(:voter).class_name(User).with_foreign_key(:user_id) }
```